src/styles/CompareCards.ts

```diff
@@ -109,7 +109,7 @@ export const CompareCardsStyles = (mode: string) => {
               height: 'auto',
             },
             '.form-wrapper': {
-              padding: `${Spacing.xxLarge}px ${Spacing.xxLarge + 2}px`,
+              padding: `${Spacing.xLarge}px ${Spacing.xxLarge + 2}px`,
             },
```

Comment:
I think we want to keep the `xxLarge` on the SearchView.
One way to do it would be to have 2 different classes: `form-wrapper` and `form-wrapper-compact` for example. Then in `CompareWithBase` and `CompareOverTime` you could pick one or the other depending on the value of `hasEditButton`.

Alternatively you can remove the CSS from here and add it directly to the component, for example adding a new `<Box>` around the  `<Form>` (the Form is from react-router and can't have the `sx` prop):

```
<Box sx={{paddingBlock: hasEditButton ? 4 : 6, paddingInline: 6}}>
  <Form>
    ...
  </Form>
</Box>
```

(remember that `sx` multiplies the number with the spacing value, that is 8px)

This proposal also changes for `padding-inline` `Spacing.xxLarge + 2` (50px) to `Spacing.xxLarge` (6 * 8 = 48px), which makes more sense to me.

Reply:
Thinking about this more, I believe we should go with the second solution.

Why? Because I believe the bottom padding should be even more tweaked, with many cases that are known from within the component:

1. hasEditMode = false => bigger padding block, to give more space on the home page.
2. hasEditMode = true, formDisplayed = false => the _bottom_ padding (that is `padding-block-end`) should be larger, because the _visual_ top padding is larger, due to the edit button taking some space in addition to the real top padding. Therefore orobably 48px would be visually looking good.
3. hasEditMode = true, formDisplayed = true => the bottom padding can move back to 32px because the cancel and compare button take some space too.

Can you try this and we'll see how it looks like?